### PR TITLE
Minor NavBar and GET fixes

### LINKED ISSUE #35: Make navbar close on new page
- [x] Navigator bar stays open when moving to a different page
- [x] When reloading a page it *Cannot GET* error

### PR DESCRIPTION
**Pull request by Amirk390**
Fied made:

+  Navigator bar stays open when moving to a different page
+ When reloading a page it Cannot GET error

close #35

### CODE REVIEW

**Review by SamiSha99 — APPROVED** [review]
:100: :+1:

File: src/NavBar.jsx [hunk 1]
```diff
@@ -122,31 +122,48 @@ class NavBar extends Component {
...
           </MenuContainer>
         </Nav>
         <DropNav>
           <DropdownMenu show={this.state.menuClick} id="button">
-            <HLink to="/">Home</HLink>
-            <HLink to="/calculator">Calculator</HLink>
-            <HLink to="/prices">Prices</HLink>
-            <HLink to="/statistics">Statistics</HLink>
+            <HLink
+              onClick={() =>
```
Comment (matthewdking) [inline src/NavBar.jsx]: this should use your `handleClick` function

File: src/NavBar.jsx [hunk 2]
```diff
@@ -122,31 +122,48 @@ class NavBar extends Component {
     this.state = {
       menuClick: false,
     };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
```
Comment (matthewdking) [inline src/NavBar.jsx]: naming? what click does this handle?

**Review by matthewdking — CHANGES_REQUESTED** [review]

File: src/NavBar.jsx (hunk 1 above)
Comment (Amirk390) [reply src/NavBar.jsx]: @matthewdking 
Why? the function is working as it is now
can you please explain this?

**Review by SamiSha99 — APPROVED** [review]
Looks good for me :white_check_mark:

**Review by matthewdking — APPROVED** [review]

**Pull request merged**